mapview/source.py

```diff
@@ -30,7 +31,13 @@ class MapSource(object):
         "thunderforest-transport": (0, 0, 19, "http://{s}.tile.thunderforest.com/transport/{z}/{x}/{y}.png", attribution_thunderforest),
         "thunderforest-landscape": (0, 0, 19, "http://{s}.tile.thunderforest.com/landscape/{z}/{x}/{y}.png", attribution_thunderforest),
         "thunderforest-outdoors": (0, 0, 19, "http://{s}.tile.thunderforest.com/outdoors/{z}/{x}/{y}.png", attribution_thunderforest),
-        
+        "dark matter":(0, 0, 18, "http://{s}.basemaps.cartocdn.com/dark_all/{z}/{x}/{y}.png", cartoDB),
+        "positron":(0, 0, 18, "http://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png", cartoDB),
+        "dark matter lite":(0, 0, 18, "http://{s}.basemaps.cartocdn.com/dark_nolabels/{z}/{x}/{y}.png", cartoDB),
+        "positron lite":(0, 0, 18, "http://{s}.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png", cartoDB),
+        "CartoDB World Antique":(0, 0, 18, "https://cartocdn_{s}.global.ssl.fastly.net/base-antique/{z}/{x}/{y}.png", cartoDB),
+        "CartoDB World Eco":(0, 0, 18, "https://cartocdn_{s}.global.ssl.fastly.net/base-eco/{z}/{x}/{y}.png", cartoDB),			
```

Comment:
It would probably be best to stick with the naming convention and formatting the other layers use, e.g.:
```python
        "carto-darkmatter": (0, 0, 18, "http://{s}.basemaps.cartocdn.com/dark_all/{z}/{x}/{y}.png", cartoDB),
        "carto-positron": (0, 0, 18, "http://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png", cartoDB),
        "carto-darkmatter-nolabels": (0, 0, 18, "http://{s}.basemaps.cartocdn.com/dark_nolabels/{z}/{x}/{y}.png", cartoDB),
        "carto-positron-nolabels": (0, 0, 18, "http://{s}.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png", cartoDB),
        "carto-base-antique": (0, 0, 18, "https://cartocdn_{s}.global.ssl.fastly.net/base-antique/{z}/{x}/{y}.png", cartoDB),
        "carto-base-eco": (0, 0, 18, "https://cartocdn_{s}.global.ssl.fastly.net/base-eco/{z}/{x}/{y}.png", cartoDB),
```